dev-util/lxqt-build-tools/lxqt-build-tools-0.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Shouldn't this be 1999-2017 for new files?

Reply:
Pull requests sometimes processed so fast that I can make it 1999-2018 or even 1999-2019 (but I assume there would be other releases of lxqt before 2019) :)
I'll update it.